lib/shared/log/tasks.js

```diff
@@ -29,22 +32,23 @@ function logTasks(tree, opts, getTask) {
 
 function printTaskTree(tree, opts) {
   var lines = [];
-  lines.push({ label: tree.label });
   var maxLabelWidth = 0;
 
   tree.nodes.forEach(function(node, idx, arr) {
     var isLast = idx === arr.length - 1;
     var w = createTreeLines(node, lines, opts, 1, '', isLast);
-    maxLabelWidth = Math.max(maxLabelWidth, w || 0);
+    maxLabelWidth = Math.max(maxLabelWidth, w || /* istanbul ignore next */ 0);
   });
 
+  log.info(msgs.tasks.description, tree.label);
+
   lines.forEach(function(line) {
-    var s = line.label;
     if (line.desc) {
-      var spaces = ' '.repeat(maxLabelWidth - line.label.length) + '  ';
-      s += spaces + line.desc;
+      var spaces = ' '.repeat(maxLabelWidth - line.width) + '  ';
+      log.info(line.fmt, line.bars, line.name, true, spaces, line.desc);
```

Comment:
Do you think users will configure this?

Reply:
At least, those colors should be configurable according to console background color.